test/unit/auth/test_jwt_auth.py

```diff
@@ -25,6 +25,11 @@ def jwt_algorithm(request):
     return request.param
 
 
+@pytest.fixture(params=('key_id_1', 'key_id_2'))
```

Comment:
What benefits do we get from testing with two mostly-identical keys? Even if they were completely different, does this get us anything? In other test files, we don't e.g. test with two different object ids.